packages/frontend/src/components/table/ProjectCell.tsx

```diff
@@ -35,6 +36,16 @@ export function ProjectCell({ project, type }: ProjectCellProps) {
           src={`/icons/${project.slug}.png`}
           alt={`${project.name} logo`}
         />
+        {project.showTypeBadge && (
+          <div
+            role="img"
+            aria-label={type}
+            className="absolute -bottom-1 left-2.5 text-3xs font-bold px-0.5 rounded-sm bg-gray-800 dark:bg-gray-200 text-white dark:text-black"
+          >
+            {type === 'layer2' && 'L2'}
+            {type === 'bridge' && 'BR'}
```

Comment:
This will be always false for the current implementation, I understand the badge will also be used to indicate L2s in the future.